docs/Contributing/Deploying-chrome-test-ext.md

```diff
@@ -0,0 +1,89 @@
+# Deploying ChromeOS test Extensions to enrolled ChromeBooks
+
+As part of validating any ChromeOS extention, run this process to force-install extension on our debug Chromebooks.
+
+## Build the extension
+
+### Bump the extension version
+
+Open ee/fleetd-chrome package.json for edit.
+Modify the version field at the top of the file (Line 4 when writing this doc)
+```
+{
+  "name": "fleetd-for-chrome",
+  "description": "Extension for Fleetd on ChromeOS",
+  "version": "1.0.3",
+  ...
+```
+Bump the version also in ee/fleetd-chrome/updates.xml
+
+### Build the distribution folder
+
+```
+cd ee/fleetd-chrome
+yarn run build
+```
+
+### Pack the extension
+
+- Go to chrome web browser.
+- go to chrome://extensions
+- Press "Pack Extensions" button (Top left of the screen)
+- Press the top Browse button and select the newly created dist folder (ee/fleetd-chrome/dist)
+- Press "Pack Extension" (No need to give it a key. It will generate it)
+
+### Load the new extension to the Chrome web browser
+
+- Open the finder app 
+- Drag and srop the ee/fleetd-chrome/dist.crx binary file to chrome web browser
+- Press "Add Extension"
+- Verify that the extension works
+- **Copy the appid for later use.**
+
+
+
+## Run a local server to make the new extension available
+
+### Edit update.xml
+Open ee/fleetd-chrome/update.xml for edit and modify:
+- the version
+- the appid (What we copied from the extension in the web browser). This will only be done for debug versions. For production we will keep the original ID we have.
```

Comment:
```suggestion
- The version.
- The `appid` (copied previously). This will only be done for debug versions. For production, we will keep the original ID we have.
```